reminders.txt

```diff
@@ -18,3 +18,5 @@ Hey human, remember to have some family time (assuming you have a supportive fam
 Hey human, is there anything else you think I should learn to say? Let me know👇
 Hey human, remember to blink just enough to keep your eyes lubricated. Screens can be hypnotizing. 😵‍💫
 Hey human, how about listening to your favorite song 🎙️🎵
+Hey human, is the weather good today? Let's go for a walk!
```

Comment:
Let's change:

> Let's go for a walk!

to:

> If it is, let's go for a walk!

Reply:
This is my first open-source contribution, so I don't know if I make the modifications or if you do? (No tutorial explains that, lol)

Reply:
I think you can push your new version to your branch, and PR should be automatically updated